.github/workflows/CI.yml

```diff
@@ -3,7 +3,30 @@ name: CI
 on: [push, pull_request]
 
 jobs:
-  test:
+
+  test1:
+    name: Julia ${{ matrix.version }} - ${{ matrix.os }} - ${{ matrix.arch }} (system libhdf5)
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        version:
+          - '1.3'
+        os:
+          - ubuntu-20.04
```

Comment:
change to `ubuntu-latest` when gh-actions get's updated (required for libhdf5 v1.10.4)